mpv: 0.39.0 -> 0.40.0

SchweGELBin opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

https://github.com/mpv-player/mpv/releases/tag/v0.40.0
Added HDR support and fixed an audio bug.

I also added myself as a maintainer

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

matthiasbeyer reviewed (changes requested):
Please split the commit into two, one where you add yourself, one to update the package.

SchweGELBin commented:
> Please split the commit into two, one where you add yourself, one to update the package.

Thank you for your reply.
I will do that in about an hour.
Have a great day!

matthiasbeyer commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 393399`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 5 packages failed to build:</summary>
  <ul>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>oneanime</li>
    <li>oneanime.debug</li>
    <li>oneanime.pubcache</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 145 packages built:</summary>
  <ul>
    <li>adl</li>
    <li>ani-cli</li>
    <li>anilibria-winmaclinux</li>
    <li>anime-downloader</li>
    <li>anime-downloader.dist</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bloomeetunes</li>
    <li>bloomeetunes.debug</li>
    <li>bloomeetunes.pubcache</li>
    <li>castero</li>
    <li>castero.dist</li>
    <li>celluloid</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>delfin</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>finamp</li>
    <li>finamp.debug</li>
    <li>finamp.pubcache</li>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>harmony-music</li>
    <li>harmony-music.debug</li>
    <li>harmony-music.pubcache</li>
    <li>haruna</li>
    <li>hypnotix</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>jellyfin-tui</li>
    <li>jellyflix</li>
    <li>jellyflix.debug</li>
    <li>jellyflix.pubcache</li>
    <li>jftui</li>
    <li>kazumi</li>
    <li>kazumi.debug</li>
    <li>kazumi.pubcache</li>
    <li>kdePackages.mpvqt</li>
    <li>kdePackages.mpvqt.debug</li>
    <li>kdePackages.mpvqt.dev</li>
    <li>kdePackages.mpvqt.devtools</li>
    <li>kdePackages.plasmatube</li>
    <li>kdePackages.plasmatube.debug</li>
    <li>kdePackages.plasmatube.dev</li>
    <li>kdePackages.plasmatube.devtools</li>
    <li>kdePackages.wallpaper-engine-plugin</li>
    <li>kdePackages.wallpaper-engine-plugin.debug</li>
    <li>kdePackages.wallpaper-engine-plugin.dev</li>
    <li>kdePackages.wallpaper-engine-plugin.devtools</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>klipperscreen</li>
    <li>libsForQt5.plasmatube (plasma5Packages.plasmatube)</li>
    <li>libsForQt5.tokodon (plasma5Packages.tokodon)</li>
    <li>linux-wallpaperengine</li>
    <li>mangayomi</li>
    <li>mangayomi.debug</li>
    <li>mangayomi.pubcache</li>
    <li>memento</li>
    <li>minitube</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
    <li>mpvScripts.inhibit-gnome</li>
    <li>mpvScripts.mpris</li>
    <li>mpvpaper</li>
    <li>musicpod</li>
    <li>musicpod.debug</li>
    <li>musicpod.pubcache</li>
    <li>photoqt</li>
    <li>pilipalax</li>
    <li>pilipalax.debug</li>
    <li>pilipalax.pubcache</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plex-mpv-shim.dist</li>
    <li>previewqt</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>qimgv</li>
    <li>radio-cli</li>
    <li>radioboat</li>
    <li>rssguard</li>
    <li>simple-live-app</li>
    <li>simple-live-app.debug</li>
    <li>simple-live-app.pubcache</li>
    <li>somafm-cli</li>
    <li>spotube</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>supersonic-wayland</li>
    <li>svp</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>tsukimi</li>
    <li>vidcutter</li>
    <li>vidcutter.dist</li>
    <li>wiliwili</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>

matthiasbeyer commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 393399`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>oneanime</li>
    <li>oneanime.debug</li>
    <li>oneanime.pubcache</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 147 packages built:</summary>
  <ul>
    <li>adl</li>
    <li>ani-cli</li>
    <li>anilibria-winmaclinux</li>
    <li>anime-downloader</li>
    <li>anime-downloader.dist</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bloomeetunes</li>
    <li>bloomeetunes.debug</li>
    <li>bloomeetunes.pubcache</li>
    <li>castero</li>
    <li>castero.dist</li>
    <li>celluloid</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>delfin</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>finamp</li>
    <li>finamp.debug</li>
    <li>finamp.pubcache</li>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>harmony-music</li>
    <li>harmony-music.debug</li>
    <li>harmony-music.pubcache</li>
    <li>haruna</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>hypnotix</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>jellyfin-tui</li>
    <li>jellyflix</li>
    <li>jellyflix.debug</li>
    <li>jellyflix.pubcache</li>
    <li>jftui</li>
    <li>kazumi</li>
    <li>kazumi.debug</li>
    <li>kazumi.pubcache</li>
    <li>kdePackages.mpvqt</li>
    <li>kdePackages.mpvqt.debug</li>
    <li>kdePackages.mpvqt.dev</li>
    <li>kdePackages.mpvqt.devtools</li>
    <li>kdePackages.plasmatube</li>
    <li>kdePackages.plasmatube.debug</li>
    <li>kdePackages.plasmatube.dev</li>
    <li>kdePackages.plasmatube.devtools</li>
    <li>kdePackages.wallpaper-engine-plugin</li>
    <li>kdePackages.wallpaper-engine-plugin.debug</li>
    <li>kdePackages.wallpaper-engine-plugin.dev</li>
    <li>kdePackages.wallpaper-engine-plugin.devtools</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>klipperscreen</li>
    <li>libsForQt5.plasmatube (plasma5Packages.plasmatube)</li>
    <li>libsForQt5.tokodon (plasma5Packages.tokodon)</li>
    <li>linux-wallpaperengine</li>
    <li>mangayomi</li>
    <li>mangayomi.debug</li>
    <li>mangayomi.pubcache</li>
    <li>memento</li>
    <li>minitube</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
    <li>mpvScripts.inhibit-gnome</li>
    <li>mpvScripts.mpris</li>
    <li>mpvpaper</li>
    <li>musicpod</li>
    <li>musicpod.debug</li>
    <li>musicpod.pubcache</li>
    <li>photoqt</li>
    <li>pilipalax</li>
    <li>pilipalax.debug</li>
    <li>pilipalax.pubcache</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plex-mpv-shim.dist</li>
    <li>previewqt</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>qimgv</li>
    <li>radio-cli</li>
    <li>radioboat</li>
    <li>rssguard</li>
    <li>simple-live-app</li>
    <li>simple-live-app.debug</li>
    <li>simple-live-app.pubcache</li>
    <li>somafm-cli</li>
    <li>spotube</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>supersonic-wayland</li>
    <li>svp</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>tsukimi</li>
    <li>vidcutter</li>
    <li>vidcutter.dist</li>
    <li>wiliwili</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>

Yarny0 commented on pkgs/applications/video/mpv/default.nix:
```diff
@@ -107,16 +107,16 @@ stdenv.mkDerivation (finalAttrs: {
     ''
       substituteInPlace meson.build \
-        --replace-fail "conf_data.set_quoted('CONFIGURATION', configuration)" \
+        --replace-fail "conf_data.set_quoted('CONFIGURATION', meson.build_options())" \
```
I'm neither an mpv expert, nor a meson expert, but the reason for this change should probably be noted in the commit message.  I don't see this as a requirement, however.

SchweGELBin replied:
In 0.39.0, this part fixed an infinite build cycle (see the comment above).
As it got modified [upstream](https://github.com/mpv-player/mpv/commit/b52a67dde2bebc83f2413bd667602d837fd65019), I modified the line to fit the upstream one again.
So this fixes an infinite build cycle.

Yarny0 replied:
Maybe put that link into the commit, so if someone later finds the commit, that person can easier understand that change.

SchweGELBin replied:
Thank you for the tip. I edited the commit message. Have a great day!

Yarny0 replied:
Thank you for preparing the update.  Your work is very much appreciated.

Yarny0 reviewed (approved):
#### Review per https://github.com/NixOS/nixpkgs/tree/698214a32beb4f4c8e3942372c694f40848b360d/pkgs#package-updates

##### Reviewed points

- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package builds on *x86-64* (after cherry-picking onto [current `nixos-unstable`](https://github.com/NixOS/nixpkgs/tree/698214a32beb4f4c8e3942372c694f40848b360d))
- [ ] executables tested on *x86-64*
- [ ] all depending packages build
- [x] patches have a comment describing either the upstream URL or a reason why the patch wasn't upstreamed
- [x] patches that are remotely available are fetched rather than vendored

More (built after cherry-picking onto [current `nixos-unstable`](https://github.com/NixOS/nixpkgs/tree/698214a32beb4f4c8e3942372c694f40848b360d), used on X11 display on current NixOS 24.11):

- [x] `mpv {path-to-mpeg2-file}` works with audio and video
- [x] `mpv dvb://{channel-via-dvb-adapter}` works with audio and video
- [x] `mpv {youtube-url}` works with audio and video

##### Comments

Just a note to the author @SchweGELBin : The author name and email fields of the three commits are filled a bit inconsistently.  I'm not sure if this is intended, so I'd like to make you aware of this.

Otherweise this looks good and it works for me!

SchweGELBin commented:
> Just a note to the author @SchweGELBin : The author name and email fields of the three commits are filled a bit inconsistently. I'm not sure if this is intended, so I'd like to make you aware of this.
> 
> Otherweise this looks good and it works for me!

Shall use lowercase letters instead?

SchweGELBin commented:
I forgot to check the checkmark before: It also builds on aarch64-linux.

Yarny0 commented:
> I forgot to check the checkmark before: It also builds on aarch64-linux.

Hint: You should be able to edit the pull-request text and thereby add the checkmark.

SchweGELBin committed:
maintainers: add SchweGELBin

SchweGELBin committed:
mpv: add SchweGELBin as maintainer

SchweGELBin committed:
mpv: 0.39.0 -> 0.40.0

mpv changed the default configuration data in the following commit:
https://github.com/mpv-player/mpv/commit/b52a67dde2bebc83f2413bd667602d837fd65019
Because ot that, the part we need to patch also changed.

SchweGELBin committed:
mpv: src.rev -> src.tag

Co-authored-by: Gaétan Lepage <33058747+GaetanLepage@users.noreply.github.com>

SchweGELBin commented on pkgs/applications/video/mpv/default.nix:
```diff
@@ -309,6 +309,7 @@ stdenv.mkDerivation (finalAttrs: {
       fpletz
       globin
       ma27
+      SchweGELBin
```
I have done it now.

GaetanLepage reviewed (approved):
LGTM! I plan to merge this in the next hours.

fpletz reviewed (approved):
Also LGTM. Thanks everyone! :tada:

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 393399`

Logs: https://github.com/GaetanLepage/nixpkgs-review-gha/actions/runs/14130159449


---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 7 packages failed to build:</summary>
  <ul>
    <li>castero</li>
    <li>castero.dist</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>oneanime</li>
    <li>oneanime.debug</li>
    <li>oneanime.pubcache</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 143 packages built:</summary>
  <ul>
    <li>adl</li>
    <li>ani-cli</li>
    <li>anilibria-winmaclinux</li>
    <li>anime-downloader</li>
    <li>anime-downloader.dist</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bloomeetunes</li>
    <li>bloomeetunes.debug</li>
    <li>bloomeetunes.pubcache</li>
    <li>celluloid</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>delfin</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>finamp</li>
    <li>finamp.debug</li>
    <li>finamp.pubcache</li>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>harmony-music</li>
    <li>harmony-music.debug</li>
    <li>harmony-music.pubcache</li>
    <li>haruna</li>
    <li>hypnotix</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>jellyfin-tui</li>
    <li>jellyflix</li>
    <li>jellyflix.debug</li>
    <li>jellyflix.pubcache</li>
    <li>jftui</li>
    <li>kazumi</li>
    <li>kazumi.debug</li>
    <li>kazumi.pubcache</li>
    <li>kdePackages.mpvqt</li>
    <li>kdePackages.mpvqt.debug</li>
    <li>kdePackages.mpvqt.dev</li>
    <li>kdePackages.mpvqt.devtools</li>
    <li>kdePackages.plasmatube</li>
    <li>kdePackages.plasmatube.debug</li>
    <li>kdePackages.plasmatube.dev</li>
    <li>kdePackages.plasmatube.devtools</li>
    <li>kdePackages.wallpaper-engine-plugin</li>
    <li>kdePackages.wallpaper-engine-plugin.debug</li>
    <li>kdePackages.wallpaper-engine-plugin.dev</li>
    <li>kdePackages.wallpaper-engine-plugin.devtools</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>klipperscreen</li>
    <li>libsForQt5.plasmatube (plasma5Packages.plasmatube)</li>
    <li>libsForQt5.tokodon (plasma5Packages.tokodon)</li>
    <li>linux-wallpaperengine</li>
    <li>mangayomi</li>
    <li>mangayomi.debug</li>
    <li>mangayomi.pubcache</li>
    <li>memento</li>
    <li>minitube</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
    <li>mpvScripts.inhibit-gnome</li>
    <li>mpvScripts.mpris</li>
    <li>mpvpaper</li>
    <li>musicpod</li>
    <li>musicpod.debug</li>
    <li>musicpod.pubcache</li>
    <li>photoqt</li>
    <li>pilipalax</li>
    <li>pilipalax.debug</li>
    <li>pilipalax.pubcache</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plex-mpv-shim.dist</li>
    <li>previewqt</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>qimgv</li>
    <li>radio-cli</li>
    <li>radioboat</li>
    <li>rssguard</li>
    <li>simple-live-app</li>
    <li>simple-live-app.debug</li>
    <li>simple-live-app.pubcache</li>
    <li>somafm-cli</li>
    <li>spotube</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>supersonic-wayland</li>
    <li>svp</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>tsukimi</li>
    <li>vidcutter</li>
    <li>vidcutter.dist</li>
    <li>wiliwili</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 7 packages failed to build:</summary>
  <ul>
    <li>castero</li>
    <li>castero.dist</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>oneanime</li>
    <li>oneanime.debug</li>
    <li>oneanime.pubcache</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 140 packages built:</summary>
  <ul>
    <li>ani-cli</li>
    <li>anilibria-winmaclinux</li>
    <li>anime-downloader</li>
    <li>anime-downloader.dist</li>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>bloomeetunes</li>
    <li>bloomeetunes.debug</li>
    <li>bloomeetunes.pubcache</li>
    <li>celluloid</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>delfin</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>finamp</li>
    <li>finamp.debug</li>
    <li>finamp.pubcache</li>
    <li>flet-client-flutter</li>
    <li>flet-client-flutter.debug</li>
    <li>flet-client-flutter.pubcache</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>harmony-music</li>
    <li>harmony-music.debug</li>
    <li>harmony-music.pubcache</li>
    <li>haruna</li>
    <li>hypnotix</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-mpv-shim</li>
    <li>jellyfin-mpv-shim.dist</li>
    <li>jellyfin-tui</li>
    <li>jellyflix</li>
    <li>jellyflix.debug</li>
    <li>jellyflix.pubcache</li>
    <li>jftui</li>
    <li>kazumi</li>
    <li>kazumi.debug</li>
    <li>kazumi.pubcache</li>
    <li>kdePackages.mpvqt</li>
    <li>kdePackages.mpvqt.debug</li>
    <li>kdePackages.mpvqt.dev</li>
    <li>kdePackages.mpvqt.devtools</li>
    <li>kdePackages.plasmatube</li>
    <li>kdePackages.plasmatube.debug</li>
    <li>kdePackages.plasmatube.dev</li>
    <li>kdePackages.plasmatube.devtools</li>
    <li>kdePackages.wallpaper-engine-plugin</li>
    <li>kdePackages.wallpaper-engine-plugin.debug</li>
    <li>kdePackages.wallpaper-engine-plugin.dev</li>
    <li>kdePackages.wallpaper-engine-plugin.devtools</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>klipperscreen</li>
    <li>libsForQt5.plasmatube (plasma5Packages.plasmatube)</li>
    <li>libsForQt5.tokodon (plasma5Packages.tokodon)</li>
    <li>mangayomi</li>
    <li>mangayomi.debug</li>
    <li>mangayomi.pubcache</li>
    <li>memento</li>
    <li>minitube</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpc-qt</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
    <li>mpvScripts.inhibit-gnome</li>
    <li>mpvScripts.mpris</li>
    <li>mpvpaper</li>
    <li>musicpod</li>
    <li>musicpod.debug</li>
    <li>musicpod.pubcache</li>
    <li>photoqt</li>
    <li>pilipalax</li>
    <li>pilipalax.debug</li>
    <li>pilipalax.pubcache</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>plex-mpv-shim.dist</li>
    <li>previewqt</li>
    <li>python312Packages.flet-cli</li>
    <li>python312Packages.flet-cli.dist</li>
    <li>python312Packages.flet-desktop</li>
    <li>python312Packages.flet-desktop.dist</li>
    <li>python312Packages.flet-web</li>
    <li>python312Packages.flet-web.dist</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.flet-cli</li>
    <li>python313Packages.flet-cli.dist</li>
    <li>python313Packages.flet-desktop</li>
    <li>python313Packages.flet-desktop.dist</li>
    <li>python313Packages.flet-web</li>
    <li>python313Packages.flet-web.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>qimgv</li>
    <li>radio-cli</li>
    <li>radioboat</li>
    <li>rssguard</li>
    <li>simple-live-app</li>
    <li>simple-live-app.debug</li>
    <li>simple-live-app.pubcache</li>
    <li>somafm-cli</li>
    <li>spotube</li>
    <li>stremio</li>
    <li>sublime-music</li>
    <li>sublime-music.dist</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>supersonic-wayland</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>tsukimi</li>
    <li>vidcutter</li>
    <li>vidcutter.dist</li>
    <li>wiliwili</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>


---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 11 packages marked as broken and skipped:</summary>
  <ul>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mpc-qt</li>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 42 packages failed to build:</summary>
  <ul>
    <li>ani-cli</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-tui</li>
    <li>klipperscreen</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>previewqt</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>radio-cli</li>
    <li>somafm-cli</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
  </ul>
</details>


---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 11 packages marked as broken and skipped:</summary>
  <ul>
    <li>anki</li>
    <li>anki.dist</li>
    <li>anki.doc</li>
    <li>anki.man</li>
    <li>ki</li>
    <li>ki.dist</li>
    <li>kikoplay</li>
    <li>mnemosyne</li>
    <li>mnemosyne.dist</li>
    <li>mpc-qt</li>
    <li>ytui-music</li>
  </ul>
</details>
<details>
  <summary>:x: 42 packages failed to build:</summary>
  <ul>
    <li>ani-cli</li>
    <li>cplay-ng</li>
    <li>cplay-ng.dist</li>
    <li>curseradio</li>
    <li>curseradio.dist</li>
    <li>dmlive</li>
    <li>dra-cla</li>
    <li>ff2mpv</li>
    <li>ff2mpv-go</li>
    <li>gonic</li>
    <li>gtk-pipe-viewer</li>
    <li>gtk-pipe-viewer.devdoc</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>invidtui</li>
    <li>jellyfin-media-player</li>
    <li>jellyfin-tui</li>
    <li>klipperscreen</li>
    <li>mov-cli</li>
    <li>mov-cli.dist</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpv-unwrapped.dev</li>
    <li>mpv-unwrapped.doc</li>
    <li>mpv-unwrapped.man</li>
    <li>pipe-viewer</li>
    <li>pipe-viewer.devdoc</li>
    <li>plex-media-player</li>
    <li>previewqt</li>
    <li>python312Packages.mpv</li>
    <li>python312Packages.mpv.dist</li>
    <li>python313Packages.mpv</li>
    <li>python313Packages.mpv.dist</li>
    <li>radio-cli</li>
    <li>somafm-cli</li>
    <li>subtitleedit</li>
    <li>supersonic</li>
    <li>termusic</li>
    <li>tomato-c</li>
    <li>wtwitch</li>
    <li>youtube-tui</li>
    <li>ytfzf</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>mpvScripts.builtins.acompressor</li>
    <li>mpvScripts.builtins.autocrop</li>
    <li>mpvScripts.builtins.autodeint</li>
    <li>mpvScripts.builtins.autoload</li>
  </ul>
</details>
